README.md

```diff
@@ -28,9 +31,33 @@ export POLICY_PATH=/path/to/your/forseti-config-policies/repo
 # Generate a terraform plan.
 terraform plan --out=terraform.tfplan
 
+```
+
+### Terraform v0.11
+
+```
 # Validate the google resources the plan would create.
-terraform-validator validate --policy-path=${POLICY_PATH} ./terraform.tfplan
+terraform-validator validate --tf-version 0.11 --policy-path=${POLICY_PATH} ./terraform.tfplan
```

Comment:
I think we may want to default to TF 11 but warn that the flag is not specified, then start failing in a later iteration so we don't break people.

Reply:
make sense